tests/src/Kernel/Access/OgEntityAccessTest.php

```diff
@@ -315,10 +317,6 @@ public function testAccess() {
 
     $this->assertTrue($og_access->userAccess($this->group1, 'some_perm', $this->user3)->isAllowed());
 
-    // A member with permission to update the group. The operation edit is
-    // passed to the userAccess method.
-    $this->assertTrue($og_access->userAccess($this->group1, 'edit', $this->user4)->isAllowed());
-
```

Comment:
This one line was the only test that covers handling an entity operation through this method. Everything else is about permissions. This is a good indication that we are on the right path with this cleanup.